### ISSUE TITLE
Additional properties are not allowed ('snippetPreviewFontSize' was unexpected)

### ISSUE THREAD

**Issue by MRYingLEE** [jupytercalpoly/jupyterlab-code-snippets#250]
**Describe the bug**
An error happened when Jupyterlab was launched.

**Expected behavior**
No error should be expected in launching.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Unbuntu (Windows Subsystem for Linux)
 - Version [e.g. 22] Windows 11

**Additional context**

Anaconda3-2023.03
JupyterLab 3.6.2

[W 2023-03-28 21:08:36.720 LabApp] The extension "jupyterlab-code-snippets" is outdated.

[I 2023-03-28 21:08:36.886 LabApp] Build is up to date
[W 2023-03-28 21:08:37.449 LabApp] Failed validating settings (jupyterlab-code-snippets:snippets): Additional properties are not allowed ('snippetPreviewFontSize' was unexpected)

    Failed validating 'additionalProperties' in schema:
        {'additionalProperties': False,
         'definitions': {'snippet': {'properties': {'code': {'type': 'array'},
                                                    'description': {'type': 'string'},
                                                    'id': {'type': 'number'},
                                                    'language': {'type': 'string'},
                                                    'name': {'type': 'string'},
                                                    'tags': {'type': 'array'}},
                                     'required': ['id', 'name', 'language'],
                                     'type': 'object'}},
         'description': 'Code Snippet Manager Settings',
         'jupyter.lab.setting-icon': 'custom-ui-components:codeSnippetIcon',
         'jupyter.lab.setting-icon-label': 'Code Snippet',
         'jupyter.lab.shortcuts': [{'command': 'codeSnippet:save-as-snippet',
                                    'keys': ['Accel Shift A'],
                                    'selector': '.jp-Notebook'},
                                   {'command': 'codeSnippet:save-as-snippet',
                                    'keys': ['Accel Shift A'],
                                    'selector': '.jp-FileEditor'}],
         'properties': {'snippets': {'default': [{'code': ['import '
                                                           'matplotlib.pyplot '
                                                           'as plt',
                                                           'import numpy as np',
                                                           '',
                                                           'x = np.linspace(0, '
                                                           '10, 100)',
                                                           '',
                                                           'plt.plot(x, '
                                                           'np.sin(x))',
                                                           'plt.plot(x, '
                                                           'np.cos(x))',
                                                           '',
                                                           'plt.show()'],
                                                  'description': 'Plotting '
                                                                 'code for '
                                                                 'sine graph '
                                                                 'using '
                                                                 'matplotlib.',
                                                  'id': 0,
                                                  'language': 'Python',
                                                  'name': 'Plotting_sine',
                                                  'tags': ['plot']},
                                                 {'code': ['import numpy as np',
                                                           'import scipy',
                                                           'import pandas as '
                                                           'pd',
                                                           'import '
                                                           'matplotlib.pyplot '
                                                           'as plt',
                                                           'import seaborn as '
                                                           'sns'],
                                                  'description': 'Import '
                                                                 'useful '
                                                                 'packages for '
                                                                 'data '
                                                                 'analytics',
                                                  'id': 1,
                                                  'language': 'Python',
                                                  'name': 'Importing packages '
                                                          'for data analytics',
                                                  'tags': ['data analytics']},
                                                 {'code': ['import keras',
                                                           'import sklearn',
                                                           'import torch',
                                                           'import tensorflow '
                                                           'as tf'],
                                                  'description': 'Import '
                                                                 'useful '
                                                                 'packages for '
                                                                 'machine '
                                                                 'learning',
                                                  'id': 2,
                                                  'language': 'Python',
                                                  'name': 'Importing packages '
                                                          'for machine '
                                                          'learning',
                                                  'tags': ['machine '
                                                           'learning']}],
                                     'description': 'The list of snippets.',
                                     'items': {'$ref': '#/definitions/snippet'},
                                     'title': 'Code Snippets',
                                     'type': 'array'}},
         'title': 'Code Snippet Manager',
         'type': 'object',
         'version': '1.0.0'}

    On instance:
        {'snippetPreviewFontSize': 3}
